### DISCUSSION TITLE
파일 업로드가 멈춰서 진행이 안됩니다.

### DISCUSSION THREAD

**Discussion by oktapgame** [naverz/zepeto-studio-kor#962] [5. Q&A]
![다운로드](https://user-images.githubusercontent.com/87909430/177253718-408f64ab-c789-4db0-8cb4-15158847b4a1.png)
안녕하세요. 
파일 업로드중에 무한 로딩이 걸려 더이상 진행이 안됩니다.
다시 업로드 할려고 해도 취소도 안되고 삭제도 안되서 아무것도 못하고 있습니다.
이런 경우 파일 새로 업로드할수 있게 취소 또는 삭제하는 기능도 넣어주세요.
혹시나 몰라서 <dl_zepeto_evangelist@naverz-corp.com> 으로도 메일 보냈습니다.
확인 좀 해주세요.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요 우선 재업로드 가능하도록 처리해 두었습니다.
새로 업로드할 수 있게 취소 또는 삭제하는 기능 역시 건의 드리겠습니다. 
감사합니다.

**Reply by oktapgame** [reply]
빠른 처리 감사드립니다. ^^
새로 업로드했습니다.

**Reply by Zepeto-TechEvangelist** [reply]
다행입니다~
